src/agentscope/rag/_reader/_word_reader.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""The Word reader to read and chunk Word documents."""
+import hashlib
+from typing import Literal
+
+from ._reader_base import ReaderBase
+from ._text_reader import TextReader
+from .._document import Document
+
+
+class WordReader(ReaderBase):
+    """The Word reader that splits text into chunks by a fixed chunk size."""
+
+    def __init__(
+        self,
+        chunk_size: int = 512,
```

Comment:
Maybe add an argument "include_image" in the constructor and try to deal with the image content if it's activated.